teacher.rb

```diff
@@ -0,0 +1,9 @@
+require './person'
+
+class Teacher < Person
+  attr_accessor :specialization
```

Comment:
- [ ] I think you missed to add a constuctor here since you are inheriting the class from Person class so you should add initialize method to  be able to access all the attribute so kindly implement it you can check this link [Adding class constructor](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/class_and_object_for_irb.md#constructor-and-instance-vars)


- Constructor extends parent's constructar by adding @specialization and a parameter for it ✅♻💯🔥

Reply:
This is noted @Reem-lab  and has been corrected. A new review requested already